scripts/publish-local-to.mjs

```diff
@@ -10,108 +10,128 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { resolve, relative } from "path";
+import { resolve, relative, basename } from "path";
 import { readFileSync } from "fs";
 import {
   getSdks,
   highlightLogVars,
   logger,
+  repoRoot,
   runWithArg,
-  sh,
+  sh as realSh,
   shResult,
 } from "./script-runner.mjs";
 
-let absDependentDir;
-let dependentName;
+// for dry runs
+const fakeSh = (cmd, args, opts = {}) => {
+  const cwd = relative(repoRoot, opts.cwd || "") || "./";
+  logger.log.hl`would run ${cwd} > ${basename(cmd)} ${args.join(" ")}`;
+};
```

Comment:
Adding dry-run capability to the publish script, to see what it would do with yalc.

Reply:
This is fine. But pay attention that one more unnecessary improvement (I'm not saying it is not useful) added to the requested feature that is far behind reasonable delivery time.